composer.json

```
@@ -26,9 +26,9 @@
         "ext-libxml": "*",
         "composer-runtime-api": "^2.0",
         "phpunit/phpunit": "^10.0|^11.0|^12.0",
-        "symfony/property-access": "^5.2|^6.2|^7.0",
-        "symfony/serializer": "^5.2|^6.2|^7.0",
-        "symfony/yaml": "^5.2|^6.2|^7.0"
+        "symfony/property-access": "^5.2||^6.2||^7.0 || ^8.0",
+        "symfony/serializer": "^5.2||^6.2||^7.0 || ^8.0",
+        "symfony/yaml": "^5.2||^6.2||^7.0 || ^8.0"
```

```suggestion
        "symfony/property-access": "^5.2||^6.2||^7.0||^8.0",
        "symfony/serializer": "^5.2||^6.2||^7.0||^8.0",
        "symfony/yaml": "^5.2||^6.2||^7.0||^8.0"
```

let's stay consistent